src/main/java/hudson/plugins/sonar/utils/SonarUtils.java

```diff
@@ -149,8 +151,8 @@ public static SonarAnalysisAction addBuildInfoTo(Run<?, ?> build, String install
     return buildInfo;
   }
 
-  public static SonarAnalysisAction addBuildInfoTo(Run<?, ?> build, String installationName) throws IOException {
-    return addBuildInfoTo(build, installationName, false);
+  public static SonarAnalysisAction addBuildInfoTo(Run<?, ?> build, FilePath workspace, String installationName) throws IOException, InterruptedException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: java.io.IOException, java.lang.InterruptedException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1160)